Dockerfile

```diff
@@ -15,16 +15,25 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 FROM common AS sgx
+ARG PSW_VERSION=2.13.103.1
+ARG DCAP_VERSION=1.10.103.1
 RUN apt update && \
-    apt install -y --no-install-recommends libsgx-dcap-ql libsgx-dcap-ql-dev libsgx-dcap-quote-verify libsgx-enclave-common libsgx-urts az-dcap-client && \
+    apt install -y \
+    az-dcap-client \
+    libsgx-dcap-ql=$DCAP_VERSION-bionic1 \
+    libsgx-dcap-ql-dev=$DCAP_VERSION-bionic1 \
+    libsgx-pce-logic=$DCAP_VERSION-bionic1 \
+    libsgx-qe3-logic=$DCAP_VERSION-bionic1 \
+    libsgx-dcap-quote-verify=$DCAP_VERSION-bionic1 \
+    libsgx-enclave-common=$PSW_VERSION-bionic1 \
+    libsgx-urts=$PSW_VERSION-bionic1 \
+    libsgx-enclave-common-dev=$PSW_VERSION-bionic1 \
```

Comment:
Previously, the -dev packages were in stage sgx-dev such that they didn't get into the deploy target. Any need that you changed this?

Reply:
I took another look at the build process:
`libsgx-qe3-logic` and `libsgx-pce-logic` are required by libsgx-dcap-ql, and therefore need to be installed in the sgx step.
`libsgx-ae-qve` `libsgx-ae-pce` and `libsgx-ae-qe3` already get installed by another package from the sgx step, even without `--no-install-recommends`
`libsgx-enclave-common-dev` is not required by anything else and i have moved it back to the sgx-dev step